apps/condo/domains/ticket/tasks/reopenDeferredTickets.js

```diff
@@ -38,13 +41,40 @@ const reopenDeferredTickets = async () => {
         // TODO(DOMA-4155): Update ticket can break on getOrCreateContactByClientData
         for (const ticket of ticketsToChange) {
             try {
-                await Ticket.update(adminContext, ticket.id, {
+                const updatedData = {
                     dv: 1,
                     sender: { fingerprint: 'auto-reopen', dv: 1 },
                     executor: { disconnectAll: true },
                     assignee: { disconnectAll: true },
                     status: { connect: { id: STATUS_IDS.OPEN } },
-                })
+                }
+                const assigneeId = get(ticket, ['assignee', 'id'])
+                const executorId = get(ticket, ['executor', 'id'])
+                const organizationId = get(ticket, ['organization', 'id'])
+                if (assigneeId) {
```

Comment:
you do not need to connect assigner again as he is already connected
you can remove 
```
assignee: { disconnectAll: true },
```
if you do not need to set assignee to null

Reply:
some was changed logic

Reply:
some was changed logic